src/encoded/batch_download.py

```diff
@@ -446,7 +446,7 @@ def metadata_tsv(context, request):
 def batch_download(context, request):
     # adding extra params to get required columns
     param_list = parse_qs(request.matchdict['search_params'])
-    param_list['field'] = ['files.href', 'files.file_type', 'files.restricted']
+    param_list['field'] = ['files.href', 'files.restricted'] + [k for k, v in param_list.items() if k.startswith('files.')]
```

Comment:
Do we even need to specify .href and .restricted here?

Reply:
I wasn’t sure, but I was afraid they might have some purpose not obvious to me, since they didn’t _appear_ to be used before either.